### PR TITLE
feat(HACBS-2127): mark new releases as automated

### PR DESCRIPTION
**Pull request by johnbieren**
As part of the release service's efforts to track which user created a release, the integration service should mark releases that it creates as automated. This information lets the release service know that a user did not manually created it. Automated is set both as a label and in the status to prevent users from falsely marking a release as automated.

### CODE REVIEW

File: controllers/snapshot/snapshot_adapter.go [hunk 1]
```diff
@@ -398,6 +398,14 @@ func (a *Adapter) createMissingReleasesForReleasePlans(application *applicationa
 			}
 			a.logger.LogAuditEvent("Created a new Release", newRelease, h.LogActionAdd,
```
[the remark this answers is not in the document]
Comment (dirgim) [reply controllers/snapshot/snapshot_adapter.go:399]: I think that the existing test update should be good enough to cover the change.

**Comment by johnbieren** [comment]
@dirgim now that code freeze is lifted, can I get some eyes on this?

**Review by dirgim — APPROVED** [review]
LGTM

File: release/releaseplan_test.go [hunk 2]
```diff
@@ -74,9 +74,10 @@ var _ = Describe("Release functions for managing Releases", Ordered, func() {
 		Expect(err == nil || errors.IsNotFound(err)).To(BeTrue())
 	})
 
-	It("ensures the Release can be created for ReleasePlan", func() {
+	It("ensures the Release can be created for ReleasePlan and is labelled as automated", func() {
```
[the remark this answers is not in the document]
Comment (johnbieren) [reply release/releaseplan_test.go:77]: :sunglasses:

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**